tests/lib/Persistence/Legacy/Content/ContentHandlerTest.php

```diff
@@ -811,6 +811,37 @@ public function testUpdateMetadataUpdatesPathIdentificationString()
         );
     }
 
+    /**
+     * @throws \Ibexa\Core\Base\Exceptions\NotFoundException
```

Comment:
We should never report internal core exception, but a contract one
```suggestion
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
```
Those are tests, so it doesn't matter much, but it's about good habits and consistency (codebase has technical debt on that later one ;) )